### PR TITLE
#879: Package updates

### LINKED ISSUE #879: Package update
### On Ubuntu 20.04
```
locales|2.31-0ubuntu9.14|2.31-0ubuntu9.15
```

### On Ubuntu 22.04
```
locales|2.35-0ubuntu3.6|2.35-0ubuntu3.7
```

### PR DESCRIPTION
**Pull request by tomuben**
fixes #879
1. On Ubuntu 20.04: locales|2.31-0ubuntu9.14|2.31-0ubuntu9.15
2. On Ubuntu 22.04: locales|2.35-0ubuntu3.6|2.35-0ubuntu3.7
3. Ignore some (kernel related) CVE's:
  - CVE-2024-26583
  - CVE-2024-26828
  - CVE-2024-26865
4. For R dependencies: Needed to pin packages `Matrix` and `MASS` to older versions which are still compatible with R 4.3

### All Submissions:

* [x] Check if your pull request goes to the correct bash branch (develop or master)?
* [x] Is the title of the Pull Request correct?
* [x] Is the title of the corresponding issue correct?
* [x] Have you updated the changelog?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../../../pulls) for the same update/change?  
* [x] Are you mentioning the issue which this PullRequest fixes ("Fixes...")

### When integrating to Develop branch:

1. [x] Remember to merge with "Squash and Merge"

### When integrating to Main branch:

1. [ ] Remember to merge with "Merge"

### CODE REVIEW

File: flavors/standard-EXASOL-8.0.0/flavor_base/flavor_base_deps_r/packages/cran_packages [hunk 1]
```diff
@@ -1,3 +1,5 @@
+MASS|7.3-58
```
Comment (tkilias) [inline flavors/standard-EXASOL-8.0.0/flavor_base/flavor_base_deps_r/packages/cran_packages:1]: Where does this come from?

Comment (tomuben) [reply flavors/standard-EXASOL-8.0.0/flavor_base/flavor_base_deps_r/packages/cran_packages:1]: I updated the PR description:

> Needed to pin packages `Matrix` and `MASS` to older versions which are still compatible with R 4.3

The latest versions of those packages require R 4.4.

Comment (tkilias) [reply flavors/standard-EXASOL-8.0.0/flavor_base/flavor_base_deps_r/packages/cran_packages:1]: can you add this as comment to the package files

Comment (tomuben) [reply flavors/standard-EXASOL-8.0.0/flavor_base/flavor_base_deps_r/packages/cran_packages]: Done

**Review by tkilias — APPROVED** [review]

**Pull request merged**